go-controller/pkg/ovn/subnetallocator/allocator_test.go

```diff
@@ -8,9 +8,9 @@ import (
 	ovntest "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing"
 )
 
-func newSubnetAllocator(clusterCIDR string, hostBits uint32) (*SubnetAllocator, error) {
+func newSubnetAllocator(clusterCIDR string, hostSubnetLen int) (*SubnetAllocator, error) {
```

Comment:
nit: kind of wish this was called hostSubnetMask instead of hostSubnetLen, I think it is less ambiguous

Reply:
Hm... I feel like `hostSubnetMask` would imply that it was a `net.IPMask`.

Anyway, it comes from a struct field that is called `HostSubnetLength`, so this is consistent with usage elsewhere in ovn-kube.